core-web/apps/dotcms-ui/src/app/app-routing.module.ts

```diff
@@ -139,6 +139,9 @@ const PORTLETS_ANGULAR: Route[] = [
     {
         canActivate: [editContentGuard],
         path: 'content',
+        data: {
+            reuseRoute: false
```

Comment:
this is necessary?

Reply:
yes, otherwise the route do not get updated.